dobi.yaml

```diff
@@ -0,0 +1,93 @@
+
+meta:
```

Comment:
There seems to be an overlap between the use/functionnality of fabric, dobi and docker-compose. Would it be a good idea to replace the use of dobi by a fabric task which would use docker-compose?